help/sources/tutorials/api/create/customer-success/zendesk.md

```diff
@@ -0,0 +1,1159 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+solution: Experience Platform
+title: Documentation self-service template
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to Zendesk using the Flow Service API.
+hide: true
+hidefromtoc: true
+exl-id: 
+---
+# Create a Zendesk connection using the Flow Service API
+
+## Overview
+
+[Zendesk](https://www.zendesk.com) is a popular customer service solution and sales tool.
+
+This [!DNL Adobe Experience Platform] [!DNL sources connector](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [!DNL Zendesk Search API > Export Search Results](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#export-search-results) that returns users information into Experience Platform from Zendesk for further processing.
+
+Zendesk uses bearer tokens as an authentication mechanism to communicate with the Zendesk API.
+
+## Prerequisites
+
+The following items are required before you start configuring the extension:
+
+* You need to have a Zendesk Support account. 
+    * Go to the Zendesk [register](https://www.zendesk.com/register/) page to register and create a Zendesk account, if you do not have one already.
+
+### Gather required credentials
+
+In order to connect Zendesk to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| subdomain | Unique domain specific to your account created during the registration process. | *xxxxx.zendesk.com*|
+| API token | Zendesk API token.<br></br>Navigate to the Zendesk website and access **[!UICONTROL Settings]** > **[!UICONTROL Apps and Integrations]** > **[!UICONTROL Zendesk API]** page. Then Navigate to the **[!UICONTROL API Tokens]** section. |*0lZnClEvkJSTQ7olGLl7PMhVq99gu26GTbJtf*|
+
+![Zendesk domain](../../../../images/tutorials/create/zendesk/zendesk-domain.png?lang=en)
```

Comment:
Like the UI doc, we can remove this domain screenshot and just add a link to "Zendesk website" in the screenshot